src/providers/ogr/qgsogrfeatureiterator.h

```diff
@@ -33,7 +33,6 @@ class QgsOgrFeatureSource : public QgsAbstractFeatureSource
     virtual QgsFeatureIterator getFeatures( const QgsFeatureRequest &request ) override;
 
   protected:
```

Comment:
Unrelated, just because it's visible in the diff: is ogrfeatureiterator intended to be subclassed?